src/Model/ModelBase.php

```diff
@@ -62,7 +62,10 @@ function __get($param) {
             );
         }
 
-        return $this->data[$param];
+        if (isset($this->data[$param]))
+            return $this->data[$param];
+        else
+            return false;
```

Comment:
Return `null` - would be better I think. Otherwise you get ambiguity if you are looking for a boolean field. And in general I think `string|null` as a dual-type is better than `string|false`

Reply:
See Commit 2b46e7a81d794af8c311c48ad4188bf9999df2be